tests/settings.py

```diff
@@ -85,6 +86,8 @@
 # during development only
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
+OPENWISP_REST_SWAGGER = True
```

Comment:
Is this necessary? The setting defaults to True so this here is redundant.

Reply:
oh, right